libs/application-generic/src/usecases/message-template/create-message-template/create-message-template.usecase.ts

```diff
@@ -24,54 +15,45 @@ export class CreateMessageTemplate {
     private messageTemplateRepository: MessageTemplateRepository,
     private layoutRepository: LayoutRepository,
     private createChange: CreateChange,
-    private updateChange: UpdateChange,
+    private updateChange: UpdateChange
   ) {}
 
-  async execute(
-    command: CreateMessageTemplateCommand,
-  ): Promise<MessageTemplateEntity> {
+  async execute(command: CreateMessageTemplateCommand): Promise<MessageTemplateEntity> {
     if ((command?.cta?.action as IMessageAction | undefined | '') === '') {
       throw new ApiException('Please provide a valid CTA action');
     }
 
     let layoutId: string | undefined | null;
     if (command.type === StepTypeEnum.EMAIL && !command.layoutId) {
-      const defaultLayout = await this.layoutRepository.findDefault(
-        command.environmentId,
-        command.organizationId,
-      );
+      const defaultLayout = await this.layoutRepository.findDefault(command.environmentId, command.organizationId);
       layoutId = defaultLayout?._id;
     } else {
       layoutId = command.layoutId;
     }
 
-    let item: MessageTemplateEntity =
-      await this.messageTemplateRepository.create({
-        cta: command.cta,
-        name: command.name,
-        variables: command.variables
-          ? normalizeVariantDefault(command.variables)
-          : undefined,
-        content:
-          command.contentType === 'editor'
-            ? sanitizeMessageContent(command.content)
-            : command.content,
-        contentType: command.contentType,
-        subject: command.subject,
-        title: command.title,
-        type: command.type,
-        _feedId: command.feedId ? command.feedId : null,
-        _layoutId: layoutId,
-        _organizationId: command.organizationId,
-        _environmentId: command.environmentId,
-        _creatorId: command.userId,
-        preheader: command.preheader,
-        senderName: command.senderName,
-        controls: command.controls,
-        output: command.output,
-        actor: command.actor,
-        code: command.code,
-      });
+    const shouldSanitize = command.contentType === 'editor' && command.type !== StepTypeEnum.CHAT;
```

Comment:
What does `contentType !== 'editor'` mean in this context? Are these use cases used by V2 in any of the flows?

Reply:
Please add a comment about why we excluded chat from the sanitization process.

Reply:
the 'editor' is a defualt contentType, the other type is 'html' editor. it is related only to v0.
excluding chat was a temporary solution to unblock users.